### PR TITLE
jwt y login

### PR DESCRIPTION
**Pull request by daryo918**

### CODE REVIEW

**Review by Nicolas000 — APPROVED** [review]

File: app/src/controllers/index.controller.ts [hunk 1]
```diff
@@ -19,24 +21,51 @@ export const getUsers=async (req: Request,resp: Response): Promise<Response> =>
 
 }
 
+
```
Comment (esteban027) [inline app/src/controllers/index.controller.ts]: Please leave only 1 space before the function.

File: app/src/controllers/index.controller.ts [hunk 2]
```diff
@@ -19,24 +21,51 @@ export const getUsers=async (req: Request,resp: Response): Promise<Response> =>
...
         "message":"creado",
         "user":{
             "name":name,
             "email":email
         }
     })
+
+}
+
+
```
Comment (esteban027) [inline app/src/controllers/index.controller.ts]: please leave only 1 space before

File: app/src/index.ts [hunk 3]
```diff
@@ -5,6 +5,7 @@ const app=express();
 
 import indexRoutes  from './routes/index';
 
+
```
Comment (esteban027) [inline app/src/index.ts]: Leave only 1 space.

File: app/src/libs/validateToken.ts [hunk 4]
```diff
@@ -0,0 +1,20 @@
+import {Request,Response,NextFunction} from "express";
+import  jwt  from "jsonwebtoken";
+
+
```
Comment (esteban027) [inline app/src/libs/validateToken.ts]: please leave only 1 space.

**Review by esteban027 — CHANGES_REQUESTED** [review]

**Review by esteban027 — APPROVED** [review]

**Pull request merged**